pkg/cmd/serviceaccount/resetcredentials/reset_credentials.go

```diff
@@ -158,9 +158,15 @@ func runResetCredentials(opts *options) (err error) {
 
 	opts.Logger.Info(opts.localizer.MustLocalize("serviceAccount.resetCredentials.log.info.resetSuccess", localize.NewEntry("Name", updatedServiceAccount.GetName())))
 
+	cfg, err := opts.Config.Load()
+	if err != nil {
+		return err
+	}
+
 	creds := &credentials.Credentials{
 		ClientID:     updatedServiceAccount.GetClientId(),
 		ClientSecret: updatedServiceAccount.GetClientSecret(),
+		TokenURL:     cfg.MasAuthURL + "/protocol/openid-connect/token",
```

Comment:
```suggestion
		TokenURL:     cfg.MasAuthURL + serviceaccount.TOKEN_CONSTANT,
```

Reply:
Yeah, realistically we should not be concatenating the URL like this, rather use something like `ResolveReference` (or something else), in case there are extra characters.

https://github.com/redhat-developer/app-services-cli/blob/db1abd96389bad334014e3cce98f210e780ac1a1/pkg/api/rbac/api.go#L130

Just wanted a quick fix so I copy/pasted it for now :)